app/controllers/rentals_controller.rb

```diff
@@ -0,0 +1,48 @@
+class RentalsController < ApplicationController
+
+  def check_out
+    customer = Customer.find_by(id: params[:customer_id])
+    if customer.nil?
+      render json: { ok: false, errors: "invalid customer id"}, status: :not_found
+      return
+    end
+    movie = Movie.find_by(id: params[:movie_id])
+    if movie.nil?
+      render json: { ok: false, errors: "invalid movid id"}, status: :not_found
+      return
+    end
+    if movie.available_inventory < 1
+      render json: { ok: false, errors: "movie not in inventory"}, status: :not_found
+      return
+    end
+    rental = Rental.new(rental_params)
+    rental.checkout_date = Date.today
```

Comment:
You're doing business logic in the controllers that should be done in model methods.